### PR TITLE
Fix to only render correct catalog items

### LINKED ISSUE #657: Using browser Find will find Catalog Items not being displayed
The patternfly react CatalogTileViewCategory component implementation caused the tiles to be rendered but not visible. This meant the browser Find was still able to find those items. Fix for the CatalogTileViewCategory is in patternfly-react-extensions version 2.9.1

To fix this issue the catalog view needs to pass CatalogTile components to the CatalogTileViewCategory component which now work as links if an href or onClick are passed.

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
CatalogTile components should be direct descendants in the CatalogTileView component.

Updates to patternfly-react-extensions 2.9.1

Fixes #657

### CODE REVIEW

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

File: frontend/public/components/catalog-items.jsx [hunk 1]
```diff
@@ -244,15 +243,17 @@ export class CatalogTileViewPage extends React.Component {
...
-          iconImg={tileImgUrl}
-          iconClass={iconClass}
-          vendor={vendor}
-          description={tileDescription} />;
-        return href ? <Link key={uid} className="co-catalog-item-tile" to={href}>{catalogTile}</Link> : catalogTile;
+        return (
+          <CatalogTile
+            id={uid}
+            key={uid}
+            href={href}
```
Comment (spadgett) [inline frontend/public/components/catalog-items.jsx]: @rhamilto fyi, this will probably conflict with #661

**Comment by spadgett** [comment]
@jeff-phillips-18 do you want to pull the patternfly update into this PR? It should make things eaiser for @sosiouxme and we can just rebase #632 

@rebeccaalpert fyi

**Comment by jeff-phillips-18** [comment]
I can, but we'll lose the click-ability of the tiles.

**Comment by spadgett** [comment]
@jeff-phillips-18 By the way, the bot will trust you if you make your membership in the OpenShift Github organization public.

**Comment by jeff-phillips-18** [comment]
Yeah, I can bump both then. And thanks, gone public.

**Comment by spadgett** [comment]
@sosiouxme OK we should be set when you have time to update yarn offline. Thanks!

**Comment by sosiouxme** [comment]
@jeff-phillips-18 thanks, please include these changes to downstream sources file:

```
--- a/sources
+++ b/sources
@@ -1,2 +1,2 @@
 401aa1438604b8e32306fa2bd4f8e211  node-v8.9.4-headers.tar.gz
-c2e254d2462e23aed3524ab07eb99ecd  yarn-offline.tar
+4d9651a62b920a31ac8733dc63a16940  yarn-offline.tar
```

**Comment by spadgett** [comment]
@jeff-phillips-18 It looks like we're handling the links differently now... This causes the entire page to reload when a tile is clicked since we're using `a` elements instead of `Link`.

**Comment by jeff-phillips-18** [comment]
Bah, sorry @spadgett. Fixed in #665

**Comment by spadgett** [comment]
No worries, thanks for the quick PR!

**Pull request merged**